.evergreen/config.yml

```
@@ -114,23 +120,93 @@ tasks:
       - func: run prebuild
 
 buildvariants:
-  - name: ubuntu2204-64
-    display_name: 'Ubuntu 22.04 64-bit'
+  - name: ubuntu2204-64-node-16
+    display_name: 'Ubuntu 22.04 64-bit - Node 16'
```

```suggestion
    display_name: 'Ubuntu 22.04 x64 - Node 16'
```

Super minor, can we make the display_name a bit clearer about the arch since arm is 64bit too. (any variation of x86_64 is fine by me)